src/main/webapp/register.jsp

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```

Comment:
this file already exists in master. make sure when you merge that your changes are maintained and the old file is deleted.